assets/js/base/components/checkbox-control/style.scss

```diff
@@ -42,11 +47,18 @@
 				background: $input-background-dark;
 				border-color: $controls-border-dark;
 			}
+
+			&:focus {
+				outline: 2px solid $controls-border-dark;
+				outline-offset: 2px;
+			}
 		}
 	}
 
 	&.has-error {
 		color: $alert-red;
+		outline: 2px solid $alert-red;
+		outline-offset: 0.5em;
```

Comment:
Just out of curiosity, why is this value using `em` while all the other outline values are using `px`?

Reply:
I don't think we have a hard and fast rule. I saw several components use .5em for margins and spacing.